cherry-picking-from-other-forks.md

```diff
@@ -0,0 +1,69 @@
+# Cherry picking from another forks
```

Comment:
This probably makes more sense within the Issue to document what you've done. I don't think it needs to be in the repo.

Reply:
Ok.  I just deleted this.